manifest.yml

```diff
@@ -24,11 +24,18 @@ platforms:
 #   arch: amd64
 terraform_binaries:
 - name: terraform
-  version: 0.12.30
-  source: https://github.com/hashicorp/terraform/archive/v0.12.30.zip  
-- name: terraform-provider-azurerm
```

Comment:
I noticed we still have the requirement for azurerm `version = ">=2.33.0"`, but this was removed. Just making sure it was intentional and we can remove this now

Reply:
Yes, I think that was because previously HCL used to create the service instance was retained forever in the database, and an old version of the brokerpak had fix against 2.33.0 making it required for future management of that instance. But we now are turning on update of HCL from the brokerpak, so it should no longer be required to ship azurerm 2.33.0.